### PR TITLE
Set cache paths

### PR DESCRIPTION
**Pull request by thespad**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-baseimage-alpine-nginx/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

File: root/defaults/nginx/nginx.conf.sample [hunk 1]
```diff
@@ -81,6 +81,11 @@ http {
     uwsgi_temp_path /tmp/nginx-uwsgi;
     scgi_temp_path /tmp/nginx-scgi;
 
+    proxy_cache_path /tmp/nginx-proxy-cache keys_zone=swag-proxy:10m;
+    fastcgi_cache_path /tmp/nginx-fcgi-cache keys_zone=swag-fcgi:10m;
+    scgi_cache_path /tmp/nginx-scgi-cache keys_zone=swag-scgi:10m;
+    uwsgi_cache_path /tmp/nginx-uwsgi-cache keys_zone=swag-uwsgi:10m;
```
Comment (nemchik) [inline root/defaults/nginx/nginx.conf.sample:87]: Should the zone names instead be `lsio-nginx-*` instead of `swag-*`?
This change applies to all of our downstream nginx based images (ex: nextcloud, etc)

Also, are these relevant to all images? or just swag?

Comment (thespad) [reply root/defaults/nginx/nginx.conf.sample]: Yeah, copy/paste fail. The issue surfaced via swag because it uses the proxy_cache and fastcgi_cache. Not found of our images using the scgi or uwsgi caches yet but they might be out there.

**Review by nemchik — APPROVED** [review]
